include/openthread/platform/radio.h

```diff
@@ -196,6 +197,7 @@ typedef struct otRadioFrame
             uint8_t        mMaxFrameRetries;   ///< Maximum number of retries allowed after a transmission failure.
             bool           mIsARetx : 1;       ///< True if this frame is a retransmission (ignored by radio driver).
             bool           mCsmaCaEnabled : 1; ///< Set to true to enable CSMA-CA for this packet, false otherwise.
+            bool           mIsOob : 1;         ///< True if this frame is an out of band frame.
```

Comment:
Are all out of band frames of key id mode 1? I hope we can avoid introducing this flag into spinel protocol.

Reply:
Per description of function `otLinkOutOfBandTransmitRequest`,

> An Out of Band frame is one that was generated outside of OpenThread

So OT should not modify or make assumption of the content of the frame. Which means it is possible that the frame is using key ID mode 1 and OT stack should not encrypt the content again.

Reply:
As we were considering removing this feature, so if these frames are only of key id mode 1, we should avoid introducing this flag. Besides, we should also add a note the existing API that this feature maybe removed. Thoughts? @abtink @jwhui

Reply:
Agreed to use a more neutral flag name.

Reply:
@bukepo I agree and would also definitely favor removing the OOB feature later on.
A more neutral flag name sounds good to me.